src/main/java/dan200/computercraft/core/apis/HTTPRequest.java

```diff
@@ -48,37 +48,47 @@ public static URL checkURL( String urlString ) throws HTTPRequestException
         return url;
     }
 
-    public HTTPRequest( String url, final String postText, final Map<String, String> headers, boolean binary ) throws HTTPRequestException
```

Comment:
This whole file shouldn't exist - I'm pretty sure it's me mucking up a rebase at some point. It's probably best to not edit it at all and someone can delete it in the future (or just delete it now).

Reply:
Thing doesn't seem to work without it though...

Reply:
Strange. From what I can tell, all references are pointing towards `apis.http.HTTPRequest` instead of `apis.HTTPRequest`. What error do you get?

Reply:
I might be wrong, I'll have to check